core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -6,8 +6,7 @@
 
 {% block header_js %}
   {{ super() }}
-  {% if exploration_version %}
```

Comment:
@supra08, I had a discussion with other members and this case looks little complicated so let's leave it as-is for now.

Reply:
Ok, so should I continue working on this issue? Maybe I can proceed with it if you explain me the complications. @DubeySandeep

Reply:
As you can see `GLOBALS.exlorationVersion` is defined only if exploration_version will be available, for this you have to investigate why this is implemented in such way?
(Although it's not required now but you can investigate this case) Thanks! :)

Reply:
Alright, I will look into it.